content/next/index.md

```diff
@@ -214,7 +214,7 @@ Configurable and usable for PHP projects as a composer dependency or usable glob
 * [Pyhist](https://github.com/jgoodman8/pyhist): A Python utility to automagically update the package version from the git history and generate the Changelog.
 * [commitizen-tools/commitizen](https://github.com/commitizen-tools/commitizen): A Python tool built to create committing rules for projects (default: conventional commit), bump project versions, and generate changelog. Configurable and usable for both Python and non-Python project. It's highly extensible through Python.
 * [git-mkver](https://github.com/idc101/git-mkver): A tool to automatically apply semantic versioning to git repositories based on _Conventional Commits_.
-* [Conventional Commits Next Version](https://gitlab.com/DeveloperC/conventional_commits_next_version): A tooling/language agnostic utility to calculate the next Semantic Versioning based upon the _Conventional Commits_ Git commit messages since the last version.
+* [Conventional Commits Next Version](https://gitlab.com/DeveloperC/conventional_commits_next_version): A tooling and language agnostic utility to calculate the next Semantic Versioning using the _Conventional Commits_ since the prior version, with monorepo support.
```

Comment:
Perhaps:

A tooling and language agnostic utility to calculate the next Semantic Version based on the _Conventional Commits_ since the prior version. Supports monorepos.

Reply:
Updated the PR to match your suggestion, thanks 👍 I checked and semantic version should be lowercase and Semantic Versioning should be uppercase as it is the name of the specification. Want me to submit a separate PR to correct this across the website?

Reply:
> Want me to submit a separate PR to correct this across the website?

@DeveloperC286 sounds good, thank you for the contribution.